toolkit/tools/imagecustomizerapi/verity.go

```diff
@@ -5,25 +5,55 @@ package imagecustomizerapi
 
 import (
 	"fmt"
+	"regexp"
+)
+
+const (
+	VerityRootDevicePath = "/dev/mapper/root"
+	VerityRootDeviceName = "root"
+)
+
+var (
+	verityNameRegex = regexp.MustCompile("^[a-z]+$")
 )
 
 type Verity struct {
-	DataPartition    IdentifiedPartition `yaml:"dataPartition"`
-	HashPartition    IdentifiedPartition `yaml:"hashPartition"`
-	CorruptionOption CorruptionOption    `yaml:"corruptionOption"`
+	// ID is used to correlate `Verity` objects with `FileSystem` objects.
```

Comment:
You forgot aligning struct columns, perhaps:

```
Id                      string               `yaml:"id"`
Name                    string               `yaml:"name"`
DataDeviceId            string               `yaml:"dataDeviceId"`
DataDeviceMountIdType   MountIdentifierType  `yaml:"dataDeviceMountIdType"`
HashDeviceId            string               `yaml:"hashDeviceId"`
HashDeviceMountIdType   MountIdentifierType  `yaml:"hashDeviceMountIdType"`
CorruptionOption        CorruptionOption     `yaml:"corruptionOption"`
```

Reply:
The code has been formatted by the Go formatter. When you add comments in between the fields, then Go no longer aligns the values vertically.